geniza/corpus/templates/corpus/snippets/document_transcription.html

```diff
@@ -69,18 +71,32 @@ <h3>{{ image_info.shelfmark }} {{ image_info.label }}</h3>
                     </div>
                 </div>
                 <div class="transcription-panel">
-                    {% if forloop.first %}
-                        <h3>Transcription</h3>
-                        <span data-transcription-target="editionFullLabel" class="current-transcription">{{ document.digital_editions.0.display|safe }}</span>
+                    {% if image_info.not_in_document %}
+                        {# if this image isn't actually in this document, link to related documents #}
+                        <ul class="excluded-related">
+                            {% for related_document in document.related_documents %}
+                                <li>
+                                    <a href="{% url 'corpus:document' related_document.pgpid %}">
+                                        <span class="doctype">{{ related_document.type }}</span>
+                                        <span class="shelfmark">{{ related_document.shelfmark|shelfmark_wrap }}</span>
+                                    </a>
+                                </li>
+                            {% endfor %}
+                        </ul>
+                    {% else %}
+                        {% if forloop.first %}
+                            <h3>Transcription</h3>
```

Comment:
it's possible a document occurs on verso only and not recto, in which case the transcription label would never be shown — I think it needs to be outside of  / separate from the in-document check

Reply:
That actually also brings up a related problem: which chunk of the transcription is displayed on verso in that same case. Since there will only be one chunk in the transcription, it won't show up at all. One example is PGPID 4474, which you can see appears on the [wrong page in QA](https://test-geniza.cdh.princeton.edu/en/documents/4474/) and is omitted entirely on this branch.

Reply:
this was what I meant by my instructions in the issue:
> when displaying transcription content in document_transcription.html, skip non-document images (not sure how to do this in current forloop structure; if it's complicated let's discuss)

Not skip as in not display but skip as in don't count excluded images for matching/indexing purposes.

I'm not sure an easy way to do it, but we should try to come up with something simple since we know the way we're displaying transcriptions is going to change soon.